ASIM/dev/ASimTester/ASimTester.csv

```diff
@@ -534,17 +534,17 @@ EventOwner,string,Optional,ProcessEvent,,,
 EventOwner,string,Optional,RegistryEvent,,,
 EventOwner,string,Optional,UserManagement,,,
 EventOwner,string,Optional,WebSession,,,
-EventProduct,string,Mandatory,AuditEvent,Enumerated,Azure|Windows|Exchange 365|Dataminr Pulse|Vectra Cloud,
-EventProduct,string,Mandatory,Authentication,Enumerated,Service Cloud|Auth0|CloudTrail|AAD|Azure Defender for IoT|M365 Defender for Endpoint|Security Events|Okta|PostgreSQL|OpenSSH|su|sudo|Vectra Cloud|SentinelOne,
+EventProduct,string,Mandatory,AuditEvent,Enumerated,Azure|WAF|Windows|Exchange 365|Dataminr Pulse|ISE|Vectra XDR|Meraki,
+EventProduct,string,Mandatory,Authentication,Enumerated,Service Cloud|Auth0|CloudTrail|AAD|ASA|Microsoft Defender for IoT|ISE|M365 Defender for Endpoint|Meraki|Security Events|Okta|PostgreSQL|OpenSSH|su|sudo|Vectra XDR|SentinelOne|WAF,
 EventProduct,string,Mandatory,Common,,,
 EventProduct,string,Mandatory,Dhcp,,,
 EventProduct,string,Mandatory,Dns,Enumerated,Umbrella|Azure Firewall|DNS Server|Sysmon|Sysmon for Linux|ZIA DNS|NIOS|Cloud DNS|Zeek|Vectra Stream|SentinelOne,
 EventProduct,string,Mandatory,FileEvent,Enumerated,Sysmon for Linux|Sysmon|M365 Defender for Endpoint|Azure File Storage|SharePoint|OneDrive|SentinelOne,
-EventProduct,string,Mandatory,NetworkSession,Enumerated,Fortigate|IOS|SDP|Vectra Stream|NSGFlow|Fireware|VPC|Azure Defender for IoT|Azure Firewall|M365 Defender for Endpoint|Sysmon|Sysmon for Linux|Windows Firewall|WireData|ZIA Firewall|CDL|PanOS|VMConnection|Meraki MX|Zeek|Firewall|ASA|Cynerio|SentinelOne,
-EventProduct,string,Mandatory,ProcessEvent,Enumerated,M365 Defender for Endpoint|Sysmon for Linux|Sysmon|Azure Defender for IoT|Security Events,
+EventProduct,string,Mandatory,NetworkSession,Enumerated,Fortigate|IOS|ISE|SDP|Vectra Stream|NSGFlow|Fireware|VPC|Azure Defender for IoT|Azure Firewall|M365 Defender for Endpoint|Sysmon|Sysmon for Linux|Windows Firewall|WireData|ZIA Firewall|CDL|PanOS|VMConnection|Meraki|Zeek|Firewall|ASA|Cynerio|SentinelOne|WAF,
+EventProduct,string,Mandatory,ProcessEvent,Enumerated,M365 Defender for Endpoint|Sysmon for Linux|Sysmon|Azure Defender for IoT|Security Events|SentinelOne,
 EventProduct,string,Mandatory,RegistryEvent,Enumerated,M365 Defender for Endpoint|Security Events|Sysmon|Windows Event|SentinelOne,
-EventProduct,string,Mandatory,UserManagement,Enumerated,SentinelOne,
-EventProduct,string,Mandatory,WebSession,Enumerated,IIS|Squid Proxy|ZIA Proxy|Vectra Stream|PanOS|CDL|Fireware|Meraki MX|Web Security Gateway|Zeek|Dataminr Pulse,
+EventProduct,string,Mandatory,UserManagement,Enumerated,ISE|Security Events|SentinelOne,
```

Comment:
See If Security Events can be replaced with actual product names

Reply:
@anki-narravula we have following EventProduct values in Windows Events Parsers:
Audit=>'Windows'
File=> 'Security Events'
UserManagement=> 'Security Events'
Authentication=>'Security Events'
Network=> 'Windows Firewall'
Process=> 'Security Events'
Registry=> 'Security Events'

EventVendor is 'Microsoft' in all.

I think having 'Windows Security Events' as Event Product would be better but we would then need to change all parsers too.

Reply:
I this Windows OS logs the telemetry available as "Windows Logs" (below) and Applications and Service logs telemetry logged by different applications other than OS. If we need to change at existing parsers, lets discuss the impact it has for customers?
![image](https://github.com/Azure/Azure-Sentinel/assets/10404181/96c5a82d-52d2-4a47-b3f9-0b6fd91d85a9)

Reply:
@anki-narravula Based on our discussion, as 'Security Events' value is being used in most of existing parsers, we'll continue using EventProduct as 'SecurityEvent'
Also, for consistency, I'll be changing Audit, EventProduct from 'Windows' to 'Security Events'